locale/pl/LC_MESSAGES/amo.po

```diff
@@ -78,20 +76,19 @@ msgstr "Używane przez"
 
 #: src/amo/components/AddonMoreInfo.js:24
 msgid "Homepage"
-msgstr ""
+msgstr "Strona domowa"
 
 #: src/amo/components/AddonMoreInfo.js:32
 msgid "Support Site"
-msgstr ""
+msgstr "Strona pomocy"
```

Comment:
The more I think about it, the more I like “Pomoc” to describe a place (as opposed to an action, where “wspierać” is appropriate.)